terraform/gcp/modules/rekor/security_policy.tf

```diff
@@ -0,0 +1,77 @@
+/**
+ * Copyright 2022 The Sigstore Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// Enable required services for this module
+resource "google_compute_security_policy" "rekor" {
+  name    = "rekor-service-security-policy"
+  project = var.project_id
+  type    = "CLOUD_ARMOR"
+
+  rule {
+    action   = "deny(502)"
+    priority = "1"
+
+    match {
+      expr {
+        expression = "int(request.headers['content-length']) > 134217728"
+      }
+    }
+    description = "Block all incoming requests > 128MB"
```

Comment:
Will this affect any current callers sending alpine packages? Or is this limit already in place?

Reply:
same as https://github.com/sigstore/public-good-instance/blob/5c50dcf1b55537685609453b751507d136a05f29/argocd/utilities/templates/scaffold.yaml#L250